plugin-build/build.gradle.kts

```diff
@@ -90,7 +90,8 @@ dependencies {
     testImplementationAar(Libs.SQLITE)
     testImplementationAar(Libs.SQLITE_FRAMEWORK)
     testRuntimeOnly(files(androidSdkPath))
-    testRuntimeOnlyAar(Libs.SENTRY_ANDROID)
+    testImplementation(Libs.SENTRY_ANDROID)
```

Comment:
hm, I think this should be at least `testImplementationAar` because the plugin is a .jar and it doesn't know how to include .aar as a dependency. Any reason you've changed it?

Reply:
tests were failing due to `ClassNotFound` for `sentry.io.IntegrationName` after adding the `sentry-android-okhttp` dependency
after adding sentry as `testImplementation` tests were succeding again

Reply:
hm, alright!